x11-misc/qt5ct/metadata.xml

```diff
@@ -14,6 +14,7 @@
 		icons, etc.) under DE/WM without Qt5 integration.
 	</longdescription>
 	<use>
+		<flag name="dbus">Enable system tray and global menu support</flag>
```

Comment:
That's a weird flag for that kind of feature. But I guess it's upstream-stupid because it implies two different features. I would probably make it unconditional and also avoids carrying a patch.